.circleci/config.yml

```diff
@@ -4,22 +4,22 @@ version: 2
 jobs:
   test-1.13:
     docker:
-      - image: cimg/go:1.13
+      - image: cimg/go:1.16
     steps:
       - checkout
       - run: go test -short ./...
       - run: go test -short -race ./...
       - run: cp manifests/metallb.yaml manifests/metallb.yaml.prev
   lint-1.13:
```

Comment:
`lint-1.13` -> `lint` ?

Reply:
I still need to find out what these actually do 😓 
Maybe we can move to github actions